java/src/jmri/jmrit/display/layoutEditor/configurexml/LayoutTurnoutXml.java

```diff
@@ -190,21 +188,32 @@ public void load(Element element, Object o) {
         String name = element.getAttribute("ident").getValue();
         double x = 0.0;
         double y = 0.0;
-        LayoutTurnout.TurnoutType tType = LayoutTurnout.TurnoutType.RH_TURNOUT;
+
         try {
             x = element.getAttribute("xcen").getFloatValue();
             y = element.getAttribute("ycen").getFloatValue();
-            tType = tTypeEnumMap.inputFromAttribute(element.getAttribute("type"));
-        } catch (org.jdom2.DataConversionException e) {
-            log.error("failed to convert layoutturnout attribute");
+        }
+        catch (DataConversionException e) {
+            log.error("failed to convert layoutturnout x & y attributes");
+        }
+
+        LayoutTurnout.TurnoutType tType = null;
+        Attribute a = element.getAttribute("type");
+        if (a != null) {
+            tType = LayoutTurnout.TurnoutType.getValue(a.getValue());
```

Comment:
Put the mapping back, please.  Enums shouldn't have anything like a "getValue" operation. If they have it, people will use it, and much of the value of the enum is lost.

Reply:
It's only there to support loading the int values via XML. It has to be public because the XML code isn't in the same package as the LayoutEditor.

Reply:
enum.getValue() is part of the base enum implementation. (it resolves enum based on the enum name.) And they made it public. I think that you meant to refer to the getXmlValue() method.

Reply:
EnumIO use restored